### ISSUE TITLE
Error in TooltipInformationWindow.CreateOverload when it cancels data.CreateTooltipInformation

### ISSUE THREAD

**Issue by mhutch** [mono/monodevelop#4357]
The following error occurs in TooltipInformationWindow.CreateOverload when it cancels the token it passes to data.CreateTooltipInformation

```
ERROR [2018-03-28 13:07:43Z]: Error while adding overload : [CompletionData: Icon=md-reference, DisplayText=System.IO, Description=, CompletionText=System.IO, DisplayFlags=None]
System.AggregateException: One or more errors occurred. ---> System.Threading.Tasks.TaskCanceledException: A task was canceled.
   --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional (System.Boolean includeTaskCanceledExceptions) [0x00011] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/Task.cs:2164 
  at System.Threading.Tasks.Task`1[TResult].GetResultCore (System.Boolean waitCompletionNotification) [0x0002b] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/Future.cs:562 
  at System.Threading.Tasks.Task`1[TResult].get_Result () [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/Future.cs:532 
  at MonoDevelop.MSBuildEditor.PackageSearch.PackageSearchCompletionData+<>c.<CreateTooltipInformation>b__5_0 (System.Threading.Tasks.Task`1[TResult] t) [0x00000] in /Users/mikayla/code/MonoDevelop.MSBuildEditor/MonoDevelop.MSBuildEditor/PackageSearch/PackageSearchCompletionData.cs:35 
  at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2[TAntecedentResult,TResult].InnerInvoke () [0x00024] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/TaskContinuation.cs:207 
  at System.Threading.Tasks.Task.Execute () [0x00010] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/Task.cs:2509 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:152 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x00037] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:187 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:156 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:128 
  at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:357 
  at MonoDevelop.Ide.CodeCompletion.TooltipInformationWindow+<AddOverload>d__11.MoveNext () [0x00036] in /Users/builder/data/lanes/5965/ace1e107/source/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.CodeCompletion/TooltipInformationWindow.cs:87 
---> (Inner Exception #0) System.Threading.Tasks.TaskCanceledException: A task was canceled.<---
```

> VS bug [#591634](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/591634)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.407 (master)

Pull Request #4362 merged by: Lluis Sanchez
Author: mono
Commit: c53726f31a769d65c4c7956e7ccd81d6c1f7c95f (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.407 (master)

Pull Request #4362 merged by: Lluis Sanchez
Author: mono
Commit: c53726f31a769d65c4c7956e7ccd81d6c1f7c95f (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #4362 merged by: Lluis Sanchez
Author: mono
Commit: c53726f31a769d65c4c7956e7ccd81d6c1f7c95f (xamarin/monodevelop)
